qemu/tests/cfg/rng_bat.cfg

```diff
@@ -24,3 +24,8 @@
         driver_verifier_cmd = "cat /sys/devices/virtual/misc/hw_random/rng_current"
         driver_name = virtio
         rng_data_rex = "\w+"
+    variants:
+        - @default:
+        - with_host_read:
+            host_read_cmd = "dd if=/dev/urandom of=/dev/null bs=1000 count=10000000"
```

Comment:
it seems the `host_read_cmd` is not same as the cmd in the test case

Reply:
need to clean the cmd when test done, otherwise it make io stress in host

Reply:
and don't suggest to update rng_bat case any more, it's a basic test

Reply:
@hereischen 
This should be ok.
original cmd is "while true;do dd if=/dev/urandom of=/dev/null bs=1 count=10;done"
I use large input blocks instead of many small input blocks to avoid too many noisy output in the debug log.